packages/viz/src/lib/style/helpers/utils.ts

```diff
@@ -112,3 +86,27 @@ export function parseGeometryType(type: string): GeometryType {
   s = s.replace(/^\w/, c => c.toUpperCase());
   return s as GeometryType;
 }
+
+export function findIndexForBinBuckets(
+  buckets: number[],
+  featureValue: number
+) {
+  const rangeComparison = (
+    definedValue: number,
+    currentIndex: number,
+    valuesArray: number[]
+  ) =>
+    featureValue < definedValue &&
+    (currentIndex === 0 || featureValue >= valuesArray[currentIndex - 1]);
+
+  return buckets.findIndex(rangeComparison);
+}
+
+export function calculateSizeBins(nBuckets: number, sizeRange: number[]) {
```

Comment:
Why not to put this method into `Classifier`?

Reply:
Because the idea of the classifier is just to have a numeric classifier with no extra logic.